lex/slots/MovieQuote.json

```diff
@@ -29,6 +29,42 @@
         }, 
         {
             "value": "you complete me"
+        }, 
```

Comment:
Amazon Lex uses the enumeration values you provide in a slot type definition to train its machine learning models. So by adding more enumeration values we can improve the model.